pkg/pillar/zedUpload/ociutil/oci.go

```diff
@@ -136,21 +118,26 @@ func Pull(registry, repo, localFile, username, apiKey string, client *http.Clien
 	if err != nil {
 		return manifestDirect, manifestResolved, size, err
 	}
-
-	cWriter := &CustomWriter{
-		fp:        w,
-		upSize:    stats,
-		prgNotify: prgchan,
-	}
 	defer w.Close()
 
+	// get updates on downloads, convert and pass them to sendStats
+	c := make(chan v1.Update, 200)
```

Comment:
Why 200? The goroutine to read the channel should be able to keep up with a channel length of 1, no?

Reply:
Honestly? No really good reason other than that was what I put in the example that got merged into the underlying library. It definitely should be able to keep up. I will change it to something like 10. Might as well have some buffering? Thoughts?